pkg/platform/runtime/buildscript/buildscript.go

```diff
@@ -241,7 +294,11 @@ func apString(f *buildexpression.Ap) string {
 		indent  = indent
 	)
 
-	if f.Name == reqFuncName {
+	if f.Name == reqFuncName ||
+		f.Name == eqFuncName || f.Name == neFuncName ||
+		f.Name == gtFuncName || f.Name == gteFuncName ||
+		f.Name == ltFuncName || f.Name == lteFuncName ||
+		f.Name == andFuncName {
```

Comment:
nit: It might be nice to create a map, slice, or function that can encapsulate this for us. So this line becomes a simple map check, `funk.Contains` or function call.